### PR TITLE
gh-22: Added MIT LICENSE

### LINKED ISSUE #22: Add a License
Add a License

### PR DESCRIPTION
**Pull request by dinukadesilva**
this closes #22

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 99XT
```
Comment (rehrumesh) [inline LICENSE]: Copyright should be for 99X Technology

**Review by rehrumesh — CHANGES_REQUESTED** [review]
Copyright should be for 99X Technology

**Comment by rehrumesh** [comment]
@dinukadesilva Please check the comments.

**Comment by dinukadesilva** [comment]
Done

**Comment by rehrumesh** [comment]
@dinukadesilva Thanks 🥇

**Pull request merged**